scripts/common/longhorn.sh

```diff
@@ -52,3 +52,146 @@ function longhorn_install_nfs_utils_if_missing_common() {
         systemctl enable nfs-utils
     fi
 }
+
+# scale down prometheus, move all 'longhorn' PVCs to provided storage class, scale up prometheus
+# Supported storage class migrations from longhorn are: 'rook' and 'openebs'
+function longhorn_to_sc_migration() {
```

Comment:
Why we need it in the common?
Who will call this one outside of the addons/openebs outside of its install?

Reply:
migration from rook